tests/operator_tests.py

```diff
@@ -453,6 +453,33 @@ def equality_helper(a, n):
             equality_helper(ak_edge_cases // a_vect, np_edge_cases // n_vect)
             equality_helper(a_vect // ak_edge_cases, n_vect // np_edge_cases)
 
+    def test_pda_sqrt(self):
+        n = np.array([4, 16, -1, 0])
```

Comment:
we should add `np.inf` here to make sure it remains infinity. Perhaps `-np.inf`, `-0.0` too to ensure we're covering our edge cases well

Reply:
Note chapel doesn't handle `(-INFINITY)**0.5` and `(-0.0)**0.5` correctly. These cases are correctly handled by the chapel `sqrt` function. I put up https://github.com/chapel-lang/chapel/issues/20542 to track this.

We should be able to get around this on the client in the meantime with a bit of hacky code... something like
```python
ret_array = # normal stuff
ret_array[pda == -np.inf] = np.nan
ret_array[pda== -0.0] = -0.0
```

Reply:
digging in a bit deeper, it seems numpy also has variation here...
```python
>>> n = np.array([-0.0, -7, -np.inf])
>>> n**0.5
array([-0., nan, nan])

>>> np.power(n, 0.5)
array([ 0., nan, inf])

>>> np.sqrt(n)
array([-0., nan, nan])

>>> n**(1/3)
array([ 0., nan, inf])

>>> np.power(n, 1/3)
array([ 0., nan, inf])

>>> n**(1/4)
array([ 0., nan, inf])

>>> np.power(n, 1/4)
array([ 0., nan, inf])
```

Reply:
I will be adding a ticket to track this once I push the other requested changes.

Reply:
Added issue #1730